scripts/release

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
```

Comment:
I'm not sure about where these command shall run but: is there any chance that any of the `git` commands require a specific version of Git that might ahead of the one available on the machine running these scripts?

Reply:
@rhymes

> I'm not sure about where these command shall run

Whoever is staging or cutting a release, it will run on their machine. The release process will remain manual for now until we know enough about the edge cases to automate it.

> is there any chance that any of the `git` commands require a specific version of Git that might ahead of the one available on the machine running these scripts?

Shouldn't need a specific version since I didn't use any of the newer commands like `switch` or `restore`, so it doesn't need to be a recent version or anything. I actually keep forgetting they exist so I've been mostly using the same git commands I've been using since I started using git in 2009 (except not my aliases to them … which [I stole from Gary Bernhardt](https://github.com/garybernhardt/dotfiles/blob/07bbd349eb0480dc07a6514efde30b8223d18b1d/.gitconfig#L21-L63) 😄).